### DISCUSSION TITLE
레벨3 UX 스터디 시나리오 | 2022.07.01

### DISCUSSION THREAD

**Discussion by wmakerjun** [woowacourse/retrospective#22] [General]
각 조별로 한 코멘트에 조 이름과 시나리오를 작성해주세요

**Comment by cjlee38** [comment]
# [모락] - 잠실

1. 배카라는 우테코에 등교하면서 ‘오늘 데일리 마스터 누구더라??’ 라는 생각이 들어 `모락` 사이트에 접속했다.
2. 네오조 그룹을 눌러서 금일의 데일리 마스터를 확인하였더니 “어머나 나잖아" 였다.
3. 계획적인 배카라는 평소 생각해 놓았던 두 개의 주제 중 하나를 정하기 위해 `모락`의 랜덤 기능을 이용했다.
4. 해당 주제를 사용해서 재밌게 데일리는 마쳤다. `모락`의 데일리 마스터 기능을 사용해서 다음 마스터를 정했다.
5. 점심에 네오조 사람들이랑 점심을 먹으려는데 `멀리 가기는 싫고, 사람도 많고 어떻게 찾지`하다가 `모락` 의 점심 메뉴 추천 기능을 사용해서 점심 메뉴를 정했다
6. 배카라는 열심히 공부 후 퇴근했다.
7. 집에 도착한 배카라는 스터디장으로부터 다음 스터디에 참여할 것인 지에 대해 투표를 하라는 메시지를 받아서 모락에 접속해 투표를 했다.
8. 배카라는 모락에 접속해 가능한 회식 시간을 입력했다.
9. 1시간 후, 배카라는 회식 시간이 정해졌다는 알람을 받고, 편안하게 즐겁게 뿌듯하게 잠들었다.

**Reply by jurlring** [reply]
네오조 점심 언제 먹을래?

**Comment by jswith** [comment]
# F12 - 잠실

1. 오늘도 하루 종일 개발을 마친 우아한씨. 거북목이 너무 심해 목에 통증을 느낀다.
2. 코딩할 때 안좋은 자세를 고치기 위해 거치대를 찾아보기로 한 우아한씨. 하지만 장비에는 문외한인 우아한씨는 고민에 빠진다.
3. 마침 옆자리 김왼쪽씨의 노트북 거치대가 좋아보인다. 하지만 우아한씨는 김왼쪽씨와 말 한마디 섞어보지 않아 차마 물어보지 못한다.
4. 결국 인터넷으로 검색해보는 우아한씨. 하지만 불필요한 광고성 글만 나오고 원하는 거치대는 나오지 않는다.
5. 차라리 다른 개발자들이 쓰는 거치대를 찾아볼 수 있으면 좋겠다는 생각이 든 우아한씨. 마침 F12라는 서비스의 광고가 보인다. 뭔가에 홀린듯 접속한다.
6. F12에 접속한 우아한씨. 혹시 김왼쪽씨의 프로필도 있을까? 검색해본다.
7. 검색해보니 김왼쪽씨의 프로필이 보인다.
8. 김왼쪽씨의 프로필을 들어가보니 마침 아까 본 거치대 정보도 보인다. 우아한씨는 그 거치대를 구매하기로 결정한다.
9. 마침내 거치대를 구매한 우아한씨. 본인의 프로필을 꾸미고자 F12 프로필에 방금 구매한 거치대를 등록한다.
10. 어라? 이 거치대를 산 사람들이 많이 쓰는 키보드가 추천으로 뜬다. 세트로 사용하면 좋겠다는 생각이 든다.
11. 다음 월급 날이 되면 사야겠다고 위시리스트에 키보드를 추가한다. 원하는 거치대도 구매했고, 같이 쓰기 좋은 키보드도 찾았다. 정말 만족스러운 하루라는 생각이 드는 우아한씨다.

**Comment by DomMorello** [comment]
# [회고덕] - 선릉
1. 아리아나는 스터디 마지막 날 팀원들이 모두 모인 자리에서 스터디의 마지막을 축하하고 있다.
2. 스터디 기간 내내 아리아나에게 작은 불만을 품고 있던 그란데는 감정을 털어내기 위해 스터디 회고를 제안했다.
3. 어디서부터 시작할 지 고민하던 아리아나는 노트북으로 회고덕 웹사이트에  방문하였고 첫 화면에 인기 회고 템플릿이 보였다.
4. 그 중 준 코치가 공유한 회고 템플릿을 발견하였고 제목을 클릭하여 템플릿을 확인했고 해당 링크를 팀원들과 공유했다.
5. 팀원들의 의견을 참고하여 해당 템플릿을 사용하기로 했고 생성하기 버튼을 눌러 팀의 색깔에 맞게 질문을 수정하여 회고를 진행했다.
6. 회고를 통해 자신의 단점을 알게된 아리아나는 문득 지난 스터디의 회고글이 궁금해졌다.
7. 집에 돌아온 아리아나는 회고덕에서 마이페이지에 접속하여 이전에 작성한 회고글을 돌아보며 성장을 다짐했다.

**Comment by hwangstar156** [comment]
# 공식(공유해줘 너의 지식) - 선릉

1. 공식이는 미션을 하던 도중 알수없는 에러에 마주하였습니다. 
2. 그래서 공식이는 이 문제를 구글에 검색해보았습니다.
3. 하지만 구글링을 해도 미션에 관련된 것은 나오지 않았습니다.
4. 그래서 `공식` 에 접속하였습니다.
5. 공식이는 이 문제에 대하여 질문을 작성합니다.
6. 이미 문제를 겪었던 크루가 공식이가 올린 질문에 대하여 구글링에서는 미처 알수없던 정보와 해결책을 답변해주었습니다.
7. 공식이는 이 답변을 참고하여 빠르게 문제를 해결하였고 원래였으면 오래걸렸을 문제를 빠르게 해결해 pr시간을 늦지 않고 낼수 있었습니다.
8. 자기 전, 공식이는 미션에 대한 회고를 하던 중 정확히 알지못하는 이슈에 대하여 다른 크루들과 토론을 하고싶었습니다.
9. 하지만 소심한 공식이는 크루들한테 직접 물어보는것이 부담스러웠습니다.
10. 그래서 공식이는 다시 `공식` 에 접속을 하였습니다.
11. 그리고 `익명` 기능을 이용하여 토론주제를 올렸습니다.
12. 여러 크루들이 이 토론주제에 대하여 각자의 의견을 말해주었고 공식이는 이 의견들을 참고하여 이슈에 대한 자신의 생각을 정립하였습니다.
13. 침대에 누운 공식이는 고민없이 꿀잠에 들었습니다🙃

**Comment by summerlunaa** [comment]
# 달록🌙 (알록이들🌈 매트 리버 파랑 티거 나인 후디) - 선릉

## 공유 캘린더 `달록`을 사용하는 `매리파티나후`의 하루
1. 출근길 지옥철에 탑승한 매리는 ‘빅뱅의 봄여름가을겨울’을 들으며 `달록`으로 일정을 확인한다.
2. 선릉 캠퍼스에 도착한 매리. 네오조 데일리 크루들이 회식을 하자고 한다. 
3. `달록`의 ‘다른 크루 일정 보기 기능’을 통해 가능한 회식 시간을 정한다.
4. `달록`에 회식 일정을 등록한다.
5. 등록된 일정을 데일리 크루들이 바로 확인한다.
6. 아차차! 다음주가 첫 스터디! 매주 화요일 오후 3시에 JPA 스터디를 진행하기로 했다.
7. 스터디장 매리는 `달록`에서 ‘매리의 JPA 스터디' 카테고리를 생성하고 초대 링크를 전송한다.
8. 스터디원들이 ‘매리의 JPA 스터디' 카테고리를 구독한다.
9. 매리는 `달록`의 ‘일정 반복 기능’을 통해 매주 화요일 오후 3시로 스터디 일정을 등록한다.
10. 퇴근길 지옥철에 탑승한 매리는 옥상달빛의 수고했어오늘도를 들으며 `달록`으로 오늘의 일정을 되돌아보며 “오늘도 갓생 살았다"라고 생각하며 알찬 하루를 마무리한다.

**Comment by kth990303** [comment]
# [내편] (알렉스 케이 제로 도리 소피아 승팡) - 선릉
1. 갑자기 소나기가 쏟아지고 있었습니다. 선릉캠퍼스에서 페어 프로그래밍을 마치고 집에 가려는 춘식이는 우산이 없어 어쩔 줄 몰라하고 있었습니다.  
2. 춘식이와 같이 취준을 하고 있는 철수가 춘식이에게 말을 걸었습니다. "춘식씨, 왜 집에 안가고 계세요?". 춘식이는 우산이 없어 눈물을 흘리며 "집에 가고 싶어요...." 라고 답했습니다.
3. 그 때, 철수가 우산을 꺼내며 말했습니다. "저랑 같이 우산 쓰고 가실래요? 😄".
4. 그 말을 듣고, 춘식이는 볼이 빨개지며 기쁜 표정으로 철수의 어깨로 몸을 기댔습니다. 철수의 도움 덕분에 춘식이는 비를 맞지 않고 무사히 집에 돌아갔습니다. 
5. 씻고 침대에 누운 춘식이는 철수에게 고마움을 표현하고 싶어졌습니다. 하지만, 철수의 연락처가 없었던 춘식이는 어떻게 철수에게 고마움을 표현할 수 있을지 고민하게 되었습니다.
6. 그 순간 춘식이는 떠올랐습니다. 우테코에서 개발한 `내편`을 활용하기로! 춘식이는 휴대폰으로 쉽고 빠르게 `내편`에 접속했습니다. 그리고 가입돼있던 우테코 모임을 통해 철수에게 메시지를 작성했습니다.
7. 다음날, 선릉캠퍼스에서 춘식이는 철수를 다시 만났습니다. 롤링페이퍼를 봤던 철수는 춘식이의 손을 잡으며 말을 건넵니다.
8. "어제 저한테 롤링페이퍼 남겨주신 거 알림 떠서 바로 확인했어요~ ❤️  같이 커피 한 잔 하실래요?"

**Comment by usageness** [comment]
# [모모]  하리 유콩 렉스 라쿤 이프 유세지 - 선릉

## 봉봉의 이야기 🍻
1. 비가 주륵주륵 내리던 선릉의 어느 강의장.
2. 집에 가긴 아쉬웠던 봉봉은 저녁에 맥주를 함께 마실 사람을 찾기 시작했습니다.
3. 봉봉은 바로 모모 사이트에 접속하여 모집 중인 저녁 음주 모임이 있는지 확인했습니다.
4. 모집 중인 저녁 음주 모임이 없는 것을 본 봉봉은 입맛을 다시며 모임을 열었습니다.
5. 음주 카테고리를 구독해 놓은 크루들은 모임 생성 알림을 보고 참을 수 없었습니다.
6. 아쉬웠던 봉봉은 어느 순간 미소를 띄고 있었습니다.

**Comment by bugoverdose** [comment]
# [꼭꼭] 정 시지프 준찌 레오 아서 루키 - 선릉

### 쭌의 이야기
1. 꼬로나에 걸렸던 코치 “쭌”은 1주일만에 선릉 캠퍼스에 출근했다. 
2. ‘아…오랜만에 크루들과 시간을 좀 보내고 싶네…’라는 생각이 든 쭌.
3. ‘그렇다고 애들한테 다짜고짜 찾아가면 오히려 부담스러워하지는 않을까?’라는 걱정을 하는 쭌. 
4. “아! 맞다!”라고 외치며 핸드폰을 켜 “꼭꼭” 사이트에 들어간 쭌. 
5. 쭌은 본인이 그동안 **발급한 커피 쿠폰들의 목록을 조회**한다.
6. 쭌은 **본인이 현재 한가하며, 선릉 캠퍼스에 존재한다는 정보를 등록**한다. 
7. 데일리 같은 조에 있는 크루들 **“찌”, “쭌찌”, “씨찌”를 먼저 찔러보며**, 본인에게 커피 쿠폰을 사용해도 된다고 메시지를 전달한다. 
8. **알람을 확인**한 크루들 “찌”와 “씨찌”는 함께 **그룹 쿠폰 사용 요청**을 보낸다. 
9. 행복하게 빙수를 먹고 커피를 마시며 즐거운 하루를 보낸 쭌, 찌, 씨찌.

**Comment by lokba** [comment]
# [터놓고 -Ternoko] 앤지 수달 열음 록바 바니 애쉬 아놀드 ❤️

<h3>면담은 터놓고</h3>
<img src="https://user-images.githubusercontent.com/19251499/176854743-35abbded-407a-4e6c-b06a-fd4b803b8a5f.png" width="40%" />



1. 레벨 3가 시작된지 얼마 지나지 않아 면담신청을 하라는 공지가 내려왔다. 
2. 까먹고 2주가 지났고, 터놓고에서 알람이 왔다.(자니...? 힘들지 않고?)
3. 알람을 보고 아차차 싶어 터놓고 사이트에 들어갔다.
4. 반가운 코치님들의 얼굴이 보였고, 면담하고 싶은 코치를 선택했다. 
5. 어?! 포코는 오후 2시밖에 시간이 안되네?! 준의 시간을 확인해볼까?
6. 준은 프리하네?!!!!! 5시에 예약해야지.
7. 시간을 선택하니 질문지가 나왔다. "이 면담을 통해 얻고 싶은게 있나요? ----"
8. 그동안 쌓여있던 감정들을 글로 옮기고 면담 요청을 보냈다.
9. 시간이 지났고 약속된 날에 면담이 진행되었다.
10. 면담이 끝나니 감동의 눈물이 흘렀다... 근데 준이 30분을 늦었네?
11. 터놓고에 들어가서 면담 완료 피드백을 남겼다. 피드백까지 남기니 정말 만족스러운 마음이 들었다.
12. 다음 면담이 기다린다. 


## 터놓고맙습니다. 🥷

**Reply by soominsohn** [reply]
면담은 터놓고~

**Reply by her0807** [reply]
면담은 터놓고~

**Reply by wmakerjun** [reply]
면담은 터놓고~

**Comment by nan-noo** [comment]
# 🪐모아모아(MOAMOA)🪐 - 선릉
**그린론, 베루스, 병민, 짱구, 태태, 디우**

## 시나리오
1. 크루들이 모여서 자바 스터디를 어떻게 만들지 고민하고 있다. 그걸 본 김모아가 "야 너네도 **모아모아**해!"라고 말한다. "그게 모야모야?"
2. 노트북을 켜서 **모아모아**를 보여주는 김모아. 자바를 검색해서 현재 개설된 자바 스터디를 한눈에 보여준다.
3. 그 중 하나의 스터디를 들여다 본다. 스터디 진행 방식, 학습 내용, 후기가 한눈에 들어온다.
4. "와 정말 편하다~! 이 사람들은 이렇게 스터디를 했구나~~!! 이제 이걸 참고해서 스터디를 만들 수 있겠다!!"
5. 서로 슬랙에 모집글 올리기를 망설이는 순간, "개설이랑 모집 다 **모아모아**에서 할 수 있어!" 김모아가 말했다.
6. "정말?" **모아모아**의 스터디 개설하기 버튼을 눌러 손쉽게 스터디 개설을 할 수 있음에 감탄을 금치 못했다.
7. 한 달 뒤, 스터디가 끝난 크루들은 이제까지 기록한 학습내용 후기를 한눈에 볼 수 있어 뿌듯했다. "우리 성장했구나~~"

**Comment by HJ-Rich** [comment]
# [줍줍] 꼬재 리차드 봄 써머 연로그 호프

### 1. 숙제 뭐였더라

A는 즐거운 방학을 맞이했습니다.
코치가 레벨 로그 공지를 올렸습니다. 공지에는 레벨 로그 관련 구글 링크가 있었습니다.
A는 여행도 가고 즐거운 방학을 보냈습니다.

개학 전날 A는 무언가 숙제가 있었다는 느낌을 받습니다.
다급히 슬랙 공지를 확인했습니다.
그러나 메시지는 삭제된 이후였습니다.

`6월 27일 이전에 보낸 메시지 잠금 해제하려면….`
A 는 B 크루에게 공지를 물어보았습니다.
B 크루는 줍줍을 소개해주었습니다.
A는 줍줍에 들어가 공지를 확인할 수 있었고, 레벨 로그를 무사히 해낼 수 있었습니다.

<br>

### 2. 사물함이 사라졌다…

사물함 배정이 완료되어 슬랙 공지가 올라왔습니다
A는 사물함 공지를 보고 자리를 머릿속으로만 기억했습니다

며칠 뒤 사물함을 사용하려 했는데 자리가 헷갈려서 공지를 재확인하려 했습니다
해당 공지는 이미 삭제되어 있었습니다

줍줍에 접속하여 공지를 재확인하고 사물함을 사용할 수 있게 되었습니다

<br>

### 3. 그땐 필요 없었는데…

슬랙 채널에 마감 체크 리스트 공지가 올라왔습니다
A 크루는 칼퇴를 좋아해서 체크 리스트 공지를 확인하지 않았습니다

A 크루는 장바구니 미션을 하며 캠퍼스에 마지막까지 있게 되었습니다
A 크루는 캠퍼스에 어떤 마감을 해야 하는지 알 수 없게 되어 당황스러웠습니다

줍줍에 접속하여 공지를 재확인하고, 무사히 마감을 마칠 수 있었습니다

**Comment by jojogreen91** [comment]
## 작심이의 3일 (30초 광고)

1월 1일 작심이는 화장실에 들어갔고 거울을 보았다. 

작심이는 뱃살을 움켜쥐며 이번 한 해는 정말 꾸준히 운동을 할 것이라 다짐했다.

예전 기억을 떠올렸을 때 항상 초반에는 의지가 있어 꾸준히 운동을 하다가 한 번 안하게 되니 의지가 떨어졌다.

마침 친구 인스타에 3일 연속 헬스장 가기 성공이라는 인증 피드가 올라온 것을 보았다. 그 친구는 나와 비슷한 의지력을 가졌는데 성공한 모습을 보니 부러웠다.

친구 피드에 #trytri 라는 태그를 보고 해당 앱을 다운로드 받았다.

어플을 시작하자 마자 여러 첼린지들이 보였는데 이 중 운동 챌린지가 상위에 있는게 보였고 참가 버튼을 눌렀다.

3일 다이어트 첼린지가 시작되었고. 24시간 안에 인증샷을 제출해야 해서 헬스장에 가서 운동을 하고 인증을 했다.

어플에서 만들어준 첫날 성공 인증화면을 다운 받아서 인스타에 올리고 친구들에게 다이어트를 시작했음을 알렸다.

다음날, 비가와서 운동이 가기 싫었지만 어플리케이션에서 리마인드 알림이 와서 마음을 다잡고 운동을 하러 갔다.

3일째, 오늘만 성공하면 챌린지가 성공이라고 생각하니 의욕이 생겨서 운동을 하고 인증을 했다.

챌린지가 성공하니 축하메시지가 떳고 다음 3일을 또 도전하겠느냐는 버튼이 나타났다.

작심이: “3일 더 해볼까?”

**Comment by yhh1056** [comment]
# 땡쿠 🙏 

후니, 비녀, 후이, 라라, 스컬, 호호
---

<img width="481" alt="스크린샷 2022-07-01 오후 5 43 51" src="https://user-images.githubusercontent.com/58363663/176859279-d42814e9-7043-4335-8328-0fe7129307c8.png">


1. 철수는 비가오는 날 춘식이에게 우산을 씌어주었다. `내편`을 통해 커피약속을 잡은 춘식이는 설레면서 어떻게 약속을 잡을지 고민했다
2. 수업을 마친 춘식이는 떨리는 마음으로 철수에게 커피를 마시자고 제안한다.
3. 하지만 인기가 많은 철수는 크루들과 커피를 이미 마셔서 일정을 잡지 못했다. 춘식이는 마음이 아팠다.
4. 그 때 철수는 춘식에게 `땡쿠`로 쿠폰을 달라고 하고 일정을 잡아보자고 대답했다.
5. 그 얘기를 들은 춘식이는 우테코 강의실에서 바로 `땡쿠`로 접속하여 철수에게 감사의 커피 쿠폰을 전달한다.
6. 카카오톡으로 쿠폰 알림을 받은 철수는 춘식이에게 금요일 오후 3시로 일정 요청을 보낸다.
7. 춘식이는 캘린더를 확인 후 일정을 수락하고 금요일 오후 3시 철수와 커피를 마시고, 춘식이는 고마움을 전달한다.
8. 고마움을 전달하고 받은 춘식이와 철수의 따뜻함 온도가 1도 올라갔다.
9. 그리고 춘식이와 철수는…❤️

**Reply by wonsss** [reply]
모야모야..다음 편 주세요...

![image](https://user-images.githubusercontent.com/59413128/176863173-8dd6c662-48a8-4206-8576-1733ef810f3d.png)

**Reply by dayelop** [reply]
철수... 이렇게 생길 거였어...?

**Comment by shindong96** [comment]
## [모라고라] 아스피 포키 썬 필즈 쿤 밧드 우디 - 잠실

1. 평일 저녁 김파덕은 캠퍼스에 남아 끝내지 못한 미션을 진행하고 있습니다.
2. 김파덕은 지금 서비스에 필요한 기술에 대해 궁금증을 느끼고 있습니다.
3. 바로 뒷자석에 앉은 낯선 크루들이 해당 주제에 대해 얘기하고 있습니다.
4. 김파덕은 민망해서 끼어들지 못하고 듣고만 있습니다.
5. 그때, 한 크루가 말합니다. "야 이거 모라고라에 올려보자."
6. 슬랙에 모라고라 알림이 도착했고, 김파덕은 반가운 마음으로 모라고라 링크를 클릭합니다!
7. 김파덕도 호기심에 로그인을 하지 않고 올라온 토론 스레드를 읽습니다.
8. 그 때 마침, 좀 전에 캠퍼스에서 들으면서 "이건 말도 안되는거지!"라고 생각했던 의견이 댓글에 올라와있습니다.
9. 김파덕도  반박을 하기 위해 의견 작성 인풋창을 클릭했습니다. 그때 "로그인이 필요합니다." 모달창이 나왔습니다.
10. '귀찮게...나갈까?' 생각한 순간 소셜로그인 버튼을 발견하고 깃헙 로그인을 클릭합니다. 
11. 간편하게 로그인에 성공한 김파덕은 캠퍼스에서 하고 싶었던 말들 모두 개운하게 댓글로 표현했습니다.
12. 작성한 댓글을 다음 날 아침 다시 확인해보니 그 밑에 파덕이 생각하지 못한 의견이 많이 달려 있었고, 댓글의 추천수도 올라가있었습니다.
13. 김파덕은 뿌듯함도 느꼈지만, 동시에 새로운 관점들을 보며 내가 관심있는 주제를 제안하면 다양한 의견을 알게 될거라는 기대갑을 갖게 되었습니다.

**Comment by yeongunheo** [comment]
## [티타임] 안 아키 마루 잉 야호 코이 - 잠실
1. 아키는 허겁지겁 PR을 날리고 쓸쓸히 집에 걸어가고 있다.
2. 지하철을 탄 아키는 고민에 빠지기 시작하는데...
3. 그는 누군가의 도움이 절실히 필요해졌다.
4. 그의 뇌리를 스쳐가는 한 문장. 그래! 지금이야! 렛츠 티타임!
5. 아키는 티타임 앱을 열고 자신을 이해해줄 것 같은 코치를 발견한다.
6. 아키는 평소에 익숙하던 캘린더로 코치의 일정을 손쉽게 확인한다.
7. 아키는 자신이 원하는 날짜와 시간에 면담을 신청한다.
8. 그와 동시에 브라운에게 면담 신청 알림이 울렸고, 브라운이 면담 일정을 픽스한다.
9. 그러자 아키에게도 면담 확정 알림이 울리고, 마음의 안정감을 느낀 아키는 편안하게 잠이 든다.

**Comment by yunjin-kim** [comment]
## [Devtopia] 결 로마 페퍼 해리 릭 알린 이브 - 잠실


1. 화창한 오후 한 때, 집에서 타이핑을 치고 있는 엄지
2. 그는 취준을 시작한지 500일이 넘어가고 있는 취준생이다.
3. 길어지는 취준 생활에 우울해지는데
4. 최근에 본 코딩 테스트 결과를 확인하는 엄지..
5. `탈락`이라는 문구에 더 우울해진 엄지..
6. 우울한 마음을 친구들과 니눠보려 인스타그램을 켜는 엄지..
7. '나 코테 떨어졌다.. 코테 어케 준비하냐.. 막막해'라고 글을 작성하는 엄지..
8. 띠링~ 일림이 왔다.
9. 엄지가 확인한 댓글에는 '힘내! 근데 코테가 뭐야?', '요즘 개발자 핫하던데 좀만 더 힘내라~', 라는 말들만 달리는데.
10. 더 우울해진 엄지.. 데브토피아에 접속한다.
11. 같은 글을 작성하고 침대에 눕자.
12. 띠링~ 알림이 왔다.
13. '어디 코테 준비했었는데?', '그거 어렵더라 나도 ㅠㅠ', '어떻게 준비하고 있어? 같이 스터디 할래?' 라는 댓글이 올라와 확인하는 엄지..
14. 엄지는 공감가는 댓글들에 위로를 받았고 이 기회를 통해 스터디에 참가하게 되었다.
15. 한달 후.. 엄지는 원하던 기업에 당당히 합격했다. 장하다 엄지..
